topics/ratchet/index.md

```diff
@@ -0,0 +1,12 @@
+---
+related: php
+created_by: Chris Boden
+display_name: Ratchet
+logo: ratchet.png
+github_url: https://github.com/ratchetphp
+released: May 11, 2012
+short_description: Ratchet is a set of libraries to handle WebSockets asynchronously in PHP.
+topic: ratchet
+url: http://socketo.me/
+---
+Ratchet is a set of libraries to handle WebSockets in PHP. It provides Ratchet for serving, Pawl as a client, and RFC6455 for I/O agnostic protocol handling. 
```

Comment:
I think "I/O agnostic" should be hyphenated as a compound adjective.

I'm unfamiliar with Ratchet, so I read a little on http://socketo.me/. The bit about "create real time, bi-directional applications between clients and servers" is interesting, if you wanted to work it into this description.